<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAircraftModelFilterBar</class>
 <widget class="QFrame" name="CAircraftModelFilterBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>85</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gl_FilterBarAircraftModel">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>4</number>
   </property>
   <property name="verticalSpacing">
    <number>2</number>
   </property>
   <item row="1" column="11">
    <widget class="QLineEdit" name="le_AirlineName">
     <property name="placeholderText">
      <string>Airline name</string>
     </property>
    </widget>
   </item>
   <item row="1" column="8">
    <widget class="QCheckBox" name="cbt_ColorLiveries">
     <property name="toolTip">
      <string>Color liveries</string>
     </property>
     <property name="text">
      <string>Color liv.</string>
     </property>
     <property name="tristate">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QWidget" name="wi_Id" native="true">
     <layout class="QHBoxLayout" name="hl_Id">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="lbl_Id">
        <property name="text">
         <string>Id:</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignLeft">
       <widget class="QLineEdit" name="le_Id">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>id</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="hs_Id">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="6">
    <widget class="QLabel" name="lbl_FileName">
     <property name="text">
      <string>File:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="7" colspan="2">
    <widget class="QLineEdit" name="le_FileName">
     <property name="placeholderText">
      <string>filename e.g. FSX/SimObjects/MyAircraft</string>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QLineEdit" name="le_ModelString">
     <property name="placeholderText">
      <string>Key, i.e. model key</string>
     </property>
    </widget>
   </item>
   <item row="0" column="10">
    <widget class="QLineEdit" name="le_AircraftIcao">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>ICAO</string>
     </property>
    </widget>
   </item>
   <item row="0" column="11">
    <widget class="QLineEdit" name="le_AircraftManufacturer">
     <property name="placeholderText">
      <string>Manufacturer</string>
     </property>
    </widget>
   </item>
   <item row="1" column="10">
    <widget class="QLineEdit" name="le_AirlineIcao">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="maxLength">
      <number>4</number>
     </property>
     <property name="placeholderText">
      <string>ICAO</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="swift::gui::filters::CFilterBarButtons" name="filter_Buttons">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="9">
    <widget class="QLabel" name="lbl_Distributor">
     <property name="text">
      <string>Distributor:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="9">
    <widget class="QLabel" name="lbl_Aircraft">
     <property name="text">
      <string>Aircraft:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="10" colspan="2">
    <widget class="swift::gui::components::CDbDistributorSelectorComponent" name="comp_DistributorSelector">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QLabel" name="lbl_Model">
     <property name="text">
      <string>Model:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QCheckBox" name="cbt_Military">
     <property name="toolTip">
      <string>Military</string>
     </property>
     <property name="text">
      <string>Mil.</string>
     </property>
     <property name="tristate">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QLineEdit" name="le_LiveryCode">
     <property name="placeholderText">
      <string>Code</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QCheckBox" name="cbt_Db">
     <property name="toolTip">
      <string>DB data?</string>
     </property>
     <property name="text">
      <string>DB</string>
     </property>
     <property name="tristate">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QCheckBox" name="cbt_IncludeExclude">
     <property name="toolTip">
      <string>Included/excluded models?</string>
     </property>
     <property name="text">
      <string>Incl.</string>
     </property>
     <property name="tristate">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QLineEdit" name="le_ModelDescription">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Description</string>
     </property>
    </widget>
   </item>
   <item row="1" column="9">
    <widget class="QLabel" name="lbl_Airline">
     <property name="text">
      <string>Airline:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QLabel" name="lbl_Livery">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Livery:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="6" alignment="Qt::AlignLeft">
    <widget class="swift::gui::components::CSimulatorSelector" name="comp_SimulatorSelector">
     <property name="minimumSize">
      <size>
       <width>125</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="6">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Combined:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="7" colspan="2">
    <widget class="swift::gui::components::CAircraftCombinedTypeSelector" name="comp_CombinedType">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>swift::gui::components::CSimulatorSelector</class>
   <extends>QFrame</extends>
   <header>gui/components/simulatorselector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::filters::CFilterBarButtons</class>
   <extends>QFrame</extends>
   <header>gui/filters/filterbarbuttons.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::components::CDbDistributorSelectorComponent</class>
   <extends>QFrame</extends>
   <header>gui/components/dbdistributorselectorcomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::components::CAircraftCombinedTypeSelector</class>
   <extends>QFrame</extends>
   <header>gui/components/aircraftcombinedtypeselector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cbt_Db</tabstop>
  <tabstop>cbt_IncludeExclude</tabstop>
  <tabstop>le_ModelString</tabstop>
  <tabstop>le_ModelDescription</tabstop>
  <tabstop>le_AircraftIcao</tabstop>
  <tabstop>le_AircraftManufacturer</tabstop>
  <tabstop>le_Id</tabstop>
  <tabstop>cbt_Military</tabstop>
  <tabstop>le_LiveryCode</tabstop>
  <tabstop>cbt_ColorLiveries</tabstop>
  <tabstop>le_AirlineIcao</tabstop>
  <tabstop>le_AirlineName</tabstop>
  <tabstop>le_FileName</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
